integration/spark2/src/main/scala/org/apache/spark/sql/parser/CarbonSparkSqlParser.scala

```diff
@@ -18,6 +18,7 @@ package org.apache.spark.sql.parser
 
 import scala.collection.mutable
 
+import org.antlr.v4.runtime.tree.TerminalNode
```

Comment:
I think it is better handle these validations at precreate table listener so that main logic will be clean.

Reply:
You mean to move all validation logic to precreate table listener or just the CTAS validation?
Anyway, CTAS will be implement soon, so that check is just temporary.

Reply:
No, I mean these type of validations to move to listeners, actually we no need to add these validations to carbon layer.  If other projects who want to block these operations can add their listener and just register it. I feel these configurations are not required.